renovate.json

```diff
@@ -25,15 +16,11 @@
       "automerge": true,
       "description": "Group minor and patch updates into a single PR",
       "groupName": "dependencies",
-      "managers": [
-        "terraform",
-        "pre-commit",
-        "gomod",
-        "dockerfile"
-      ],
       "matchUpdateTypes": [
```

Comment:
I suspect it's okay to auto-merge major revisions on a template but I'm not sure.